java/src/main/java/org/rocksdb/ExportImportFilesMetaData.java

```diff
@@ -6,14 +6,11 @@
 
 package org.rocksdb;
 
-import java.util.Arrays;
-import java.util.List;
-
 /**
  * The metadata that describes a column family.
  */
 public class ExportImportFilesMetaData extends RocksObject {
-  private ExportImportFilesMetaData(final long nativeHandle) {
+  public ExportImportFilesMetaData(final long nativeHandle) {
```

Comment:
Can this still be package (default) rather than public ?

Reply:
@alanpaxton Thanks for the reminder, I have updated it